tests/CommonTests.cs

```diff
@@ -380,44 +379,51 @@ public async Task CanHandleKeysend()
                 var info = await dest.GetInfo();
                 var node = info.NodeInfoList.First();
                 
-                var amount = LightMoney.Satoshis(21);
-                var preimage = new byte[32];
-                new Random().NextBytes(preimage);
-                var paymentHash = new uint256(Hashes.SHA256(preimage));
```

Comment:
This is only required for LND, moved it to the LND `PayAsync` method. The other implementations don't even allow to defined a payment hash for keysend payments.